src/components/common/YourCompany/YourCompany.vue

```diff
@@ -609,4 +622,10 @@ export default class YourCompany extends Mixins(
     min-width: 0.5rem;
   }
 }
+
+::v-deep {
+  .invalid-section .col-sm-3:first-child label {
+    color: $app-red !important;
+  }
```

Comment:
I thought you were passing in `showErrors` prop so the component itself could style this? Or is that only for NOB and THIS is for the addresses?

Rhetorical comment. Just food for thought for later. Is it Friday yet?

Reply:
hahah very soon sir!

The show errors prop is used on the input field rules/validations specifically. 
This is just highlight the main label as red when our local UI here determines when it's invalid. 
I didn't think this red text styling could/would apply everywhere under all circumstances, so applied it conditionally from the parent here.

Reply:
> I didn't think this red text styling could/would apply everywhere under all circumstances, so applied it conditionally from the parent here.

I think you're right. Resolving this conversation.

Reply:
Actually, it makes sense for the parent to style its components how it wants.